packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -12,6 +12,16 @@ const propTypes = {
    * Content alignment. Align to start, end, or center.
    */
   align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * String that labels the content the toolbar controls.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
This prop description is vague and unclear, not guiding the consumer as best as it could.

It also should be indicated that `ariaLabel` should be used only if a visible label for the toolbar is not already present, and that `ariaLabelledBy` should be the first choice when available to use instead.

Reply:
Resolved: https://github.com/cerner/terra-core/pull/3678/commits/5052341bb4dfab157f5f4163a9b3b3de3cbd4172